server/src/fixed-asset/asset.controller.ts

```diff
@@ -26,9 +29,58 @@ export class AssetController {
   @UserRoles(Roles.MANAGER)
   async create(@Body() createAssetDto: CreateAssetDto, @Req() req: AuthenticatedRequest): Promise<AssetResponseDto> {
     const userId = req.user.sub;
+    const organizationId = req.user.organizationId;
     
-    const asset = await this.assetService.create(createAssetDto, userId);
+    const asset = await this.assetService.create(createAssetDto, userId, organizationId);
```

Comment:
You can modify createAssetDto and add two private parameters for both IDs into it and simply add it here instead of increasing numbers of method parameters.